README.md

```diff
@@ -1,24 +1,146 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<!-- TABLE OF CONTENTS -->
 
-Things you may want to cover:
+# 📗 Table of Contents
 
-* Ruby version
+- [📖 About the Project](#about-project)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* System dependencies
+<!-- PROJECT DESCRIPTION -->
 
-* Configuration
+# 📖 Curriculum <a name="about-project"></a>
 
-* Database creation
+** BlogApp ** Is a Ruby on Rails is project that will allow the user to have an account, post a blog, comment and like.
 
-* Database initialization
+## 🛠 Built With <a name="built-with"></a>
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/documentation/">Ruby</a></li>
+  </ul>
 
-* Deployment instructions
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
 
-* ...
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Back-End Ruby on Rails operations**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get the content of this project locally you need to run the following commands in your terminal:
+
+and follow these steps.
+
+### Prerequisites
+
+- IDE(code editor) like: **Vscode**, **Sublime**, etc.
+- [Git](https://www.linode.com/docs/guides/how-to-install-git-on-linux-mac-and-windows/)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+1. Download the **Zip** file or clone the repo with:
+ ```bash
+git clone git@github.com:Gdiazdiaz/App-Blog.git
+```
+2. To access cloned directory run:
+```bash
+cd curriculum 
+```
+
+### Install
+
+> To install linters and other project's dependencies run:
+
+```bash
+ bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+```bash
+ruby main.rb
+```
```

Comment:
- [ ] Kindly add the correct commands to run your app like that:

To run the project, execute the following command:
1. setup the database:
```
rails db:setup
```
2. run the rails server:
```
rails s
```